dev-docs/e2e/bare-metal-runner.md

```diff
@@ -1,5 +1,230 @@
+# How to add a bare-metal instance to the CI
+
+## Packets and updates (SNP)
+
+AMD SEV-SNP is supported by kernels newer than 6.11 (see https://www.phoronix.com/news/Linux-6.11-KVM).
+While we want to check occasionally if the latest version breaks
+Contrast, we want to use the latest LTS version.
+
+`mainline` is a tool to manage kernel installations on Ubuntu.
+First install the `mainline` package:
+```bash
+sudo add-apt-repository ppa:cappelikan/ppa
+sudo apt-get update
+sudo apt-get install mainline pkexec
+```
+Now list all available kernel versions and install the latest LTS version. To
+figure out what the latest long term version is, refer to https://kernel.org/.
+```bash
+mainline list
+mainline install <latest LTS version e.g. 6.12.17>
```

Comment:
Could we just install Ubuntu 24.10 instead?

Reply:
Probably, yes. I also just installed the kernel from the 24.10 repos before building it myself.

Reply:
I mean, I'd just always install the latest Ubuntu LTS with the latest kernel LTS. This way you can e.g. test out non LTS kernels easily. Moreover, Hetzner doesn't have a default installer for non LTS Ubuntu. So you would have to prepare an Install medium yourself.

Reply:
In the long run, I think we should be testing on distro defaults where possible, because that's what people will deploy in their data centers and we benefit from the distro's patches. Ack on the lack of installation media.